Filter/BooleanFilter.php

```diff
@@ -66,7 +66,12 @@ public function getDefaultOptions()
      */
     public function getRenderSettings()
     {
-        return array('sonata_type_filter_default', array(
+        // NEXT_MAJOR: Remove this line when drop Symfony <2.8 support
+        $type = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
```

Comment:
can we move this to a method in `abstract class Filter extends BaseFilter` ?

in this case we avoid code duplication for the default stuff